docs/useForceUpdate.md

```diff
@@ -0,0 +1,20 @@
+# `useForceUpdate`
```

Comment:
How is this hook better/different from [`useUpdate`](https://github.com/streamich/react-use/blob/master/docs/useUpdate.md)?